csrc/codegen.cpp

```diff
@@ -1038,8 +1077,8 @@ class CudaKernelGenerator : private OptOutConstDispatch {
   }
 
   void genSerialReduction(
-      const kir::TensorIndex* output,
```

Comment:
Should the `const` in this function be changed back? Similar for other functions.

Reply:
Thanks. I thought I added back.

Reply:
Done